### PR TITLE
De-duplicate contributor graph translations

### PR DESCRIPTION
**Pull request by sahinakkaya**
I was using duplicate translations and @denyskon suggested to use `%s`'s in translations [in their review](https://github.com/go-gitea/gitea/pull/29191#discussion_r1493399445). I will make use of these translations in other PRs of mine (#29210 and #29191). If we want to include the two PRs I mentioned in v1.22, we should merge this asap.

### CODE REVIEW

**Review by denyskon — APPROVED** [review]

**Comment by sahinakkaya** [comment]
I want to ask maintainers, can I merge this PR into other PR's of mine in my local and keep working? Can I assume that I will be good if this PR is eventually merged into upstream? Or should I wait until it is really merged to upstream then fetch changes?

**Review by delvh — APPROVED** [review]

**Comment by delvh** [comment]
No, both is fine.
The first one simply means that as long as this PR is not merged, these changes will be shown in the other PRs as well.

**Pull request merged**
